src/js/utils/ConfigMinifier.js

```diff
@@ -12,6 +12,7 @@ lm.utils.ConfigMinifier = function() {
 		'selectionEnabled',
 		'dimensions',
 		'borderWidth',
+		'borderGrabWidth',
```

Comment:
I think this needs to go at the end of the list in one of the blank lines: minified URLs will break if you alter the ordering here. (they get turned into single letter/number based on their order in this list).